process-cli/src/main/assembly/bin/slcore-process-cli

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
```

Comment:
I would choose the name of the script with care, as it might appear in the OS process explorer. What about `sonarlint-backend`?